src/zope/meta/config_package.py

```diff
@@ -105,6 +107,12 @@ def handle_command_line_arguments():
         default=False,
         help='Activate support for a future non-final Python version if not'
         ' already configured in .meta.toml.')
+    parser.add_argument(
+        '--oldest-python',
+        dest='oldest_python',
+        default=OLDEST_PYTHON_VERSION,
+        help='Oldest supported Python version. Defaults to'
+             f'Python {OLDEST_PYTHON_VERSION}.')
```

Comment:
```suggestion
        help='Oldest supported Python version. Defaults to: '
             f'{OLDEST_PYTHON_VERSION}.')
```
This way, the help text gives an indication on the value syntax.

Reply:
I've adopted a habit of putting the spaces on the beginning of each line, this way they're all aligned and it's really noticeable when you forget to add one:

```python
    long_string_literal=(
        "long text goes brrrrrrrr"
        " continuation line 1"
        " continuation line 2"
        " continuation line 3"
    ),
```

Reply:
Applied both changes